tests/test_cases/test_array_buses/Makefile

```diff
@@ -0,0 +1,24 @@
+TOPLEVEL_LANG ?= verilog
+
+ifneq ($(TOPLEVEL_LANG),verilog)
+
+all:
+	@echo "Skipping test due to TOPLEVEL_LANG=$(TOPLEVEL_LANG) not being verilog"
```

Comment:
In principle, we could also have a VHDL implementation of the RTL and it should also work fine.

Reply:
Yes, I have one. But it is not working with GHDL. I'll try with ModelSim and add it to the repo.

Reply:
OK, I have no way to test (no VHDL license on my ModelSim), and ghdl does not support this (in_data is not found with the array type, and I can't access element i of a std_logic_vector like this). I can push the vhd, and disable the test in the Makefile, and create a new issue. Or, I can try and make it work (but it might take some time).